packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -424,12 +435,14 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
       required: true,
       defaultProps: {
         role: 'none',
+        key: beforeKey,
```

Comment:
are you intending to use this as a react key? there should be no need to use `useId` here, react keys are only going to be relevant for the immediate parent.

Will there gbe multiple before and after elements in one virtualizer?

Reply:
This was an effort to ensure all items in the list contained a unique key, if useId is not required I'll remove as there should only ever be one before/after bookmark and it will remain in the same position in DOM.